skbio/alignment/_util.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
+
+
+def align_score(path, seqs, match=2, mismatch=-3, gap_open=-5, gap_extend=-2):
```

Comment:
There are already quite a few alignment methods in skbio, so we need to differientiate. Maybe `pairwise_align_score`?